src/status_im/ui/screens/chat/styles/animations.cljs

```diff
@@ -19,6 +19,10 @@
    :bottom           bottom
    :position         :absolute
    :elevation        2
+   :shadow-offset    {:width 0 :height 1}
+   :shadow-radius    12
+   :shadow-opacity   0.12
+   :shadow-color     "rgba(0, 0, 0)"
```

Comment:
white?

Reply:
yeah anyway i need to fix tests

Reply:
it had an alpha channel at first, but then it moved to shadow opacity, i'll fix it